doc/api/deprecations.md

```diff
@@ -92,10 +95,10 @@ changes:
     description: Runtime deprecation.
 -->
 
-Type: Runtime
+Type: End-of-Life
 
-The `_writableState.buffer` property is deprecated. Use the
-`_writableState.getBuffer()` method instead.
+The `_writableState.buffer` property was deprecated in Node.js 0.11.15 and has
+been since removed. Use the `_writableState.getBuffer()` method instead.
```

Comment:
```suggestion
been removed. Use the `_writableState.getBuffer()` method instead.
```

...although I don't think it adds much of value to the end user to include when it was deprecated.

```markdown
`_writableState.buffer()` has been removed. Please use `_writableState.getBuffer()` instead.
```

The information about when it was deprecated and when it was removed is in the YAML matter which gets rendered in our docs. There's no need to repeat it in the text (in my opinion, anyway).

Reply:
Updated the text here to add your suggestions and remove the reference to when it was deprecated. Please take another look